### ISSUE TITLE
Keyboard example not working

### ISSUE THREAD

**Issue by chenglin2003** [DeepRoboticsLab/Lite3_MotionSDK#5]
The keyboard example does not work as stated in the documentation

Compiled on Lite3 with ROS Noetic with catkin_make (gcc/g++ aarch64)

Commands
```bash
roscore
rosrun examples example_keyboard
```

The CLI shows the controls (WASD to move, U to stand up/sit down etc.) but no input is being translated for movement. Tried through the ssh interface and directly plugging in a keyboard to the Lite3 terminal

The keyboard input seems to have been registered by example_keyboard, but the input doesn't seem to be translated to Lite3 as movement commands

**Comment by yyang0802** [comment]
Do you run the code in [high_level_sim](https://github.com/DeepRoboticsLab/Lite3_Model_Control/tree/main/high_level_sim), or use other controller codes?The example_keyboard only supports to input to the example controller in [high_level_sim](https://github.com/DeepRoboticsLab/Lite3_Model_Control/tree/main/high_level_sim).

**Comment by chenglin2003** [comment]
Yes I compiled the entire high level sim package as stated in the Simulation Manual but it doesn't work

**Comment by yyang0802** [comment]
Did you press enter key? 
![image](https://github.com/user-attachments/assets/dbe896fa-55a8-4da7-a87a-1932422893ac)
